js/MVP/components/RouterLenke.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import styled from 'styled-components';
+
+const LinkStyled = styled(Link)`
+  width: fit-content;
+`;
+
+function ButtonLenke({ to, children }) {
+  return (
+    <LinkStyled to={to} className="lenke">
+      {children}
+    </LinkStyled>
+  );
+}
+
+ButtonLenke.propTypes = {
+  children: PropTypes.node,
+  to: PropTypes.string,
+};
+
+export default ButtonLenke;
```

Comment:
Hm, duplikat av components/ButtonLenke? Ser det eksporteres samme navn ButtonLenke som den andre komponenten, og de er ganske like. Kan eventuelt vurdere å slå sammen disse to komponentene, eller eventuelt rename eksportnavn av denne til RouterLenke som filen heter

Reply:
Rettet opp navn på komponenten ✅ 

Har ikke sterke formeninger men foretrekker å ha dem separert, men enklere å vedlikeholde imo? Heldigvis en liten komponent, så åpen for å slå sammen.